app/models/enrollments/individual_market/family_enrollment_renewal.rb

```diff
@@ -277,11 +277,11 @@ def dental_renewal_product
 
   # rubocop:disable Style/RedundantReturn
   def eligible_to_get_covered?(member)
-    child_relations = %w[child ward foster_child adopted_child]
-    return true unless child_relations.include?(member.family_member.relationship)
+    valid_relations_for_pediatric_dental = ["self", "child", "ward", "foster_child", "adopted_child"]
+    return true unless valid_relations_for_pediatric_dental.include?(member.family_member.relationship)
+    return true if member.family_member.relationship == "self" && member.person.age_on(member.coverage_start_on) >= 19
```

Comment:
@vkghub don't we want to return false if the member is 19 or older?

Reply:
@jacobkagon
-  If member is primary  and age is >= 19 for previous year enrollment and age is >= 19 for renewal year enrollment. Since this not a age change from 18 to 19 they are considered as eligible.
- if the change is from 18 to 19  from current year to renewal year or <19  in both the years, then we will do additional validation after line [282](https://github.com/ideacrew/enroll/pull/3547/files#diff-933c4b003febe1b7f68ac140a818b62f27be75c51da0e3ce6f33860aa0fa5208R282)